packages/backend/src/main/java/com/codurance/open_space/controller/SpaceController.java

```diff
@@ -28,4 +30,13 @@ public Space create(@RequestBody Space space) {
         return spaceRepository.save(space);
     }
 
+    @PutMapping("/{id}/facilities")
+    public void update(@PathVariable Long id, @RequestBody Facilities facilities) {
```

Comment:
I prefer the method name to be updateFacilities, or move this method into its own controller